src/content/pages/privacy.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Privacy policy
+metaDescription: We care about your privacy! Find out more about how your data is used to make Turnip better.
+---
+
+import { Link, Paragraph } from '@octopusthink/nautilus'
+
+import Panel from 'components/Panel'
+
+<Panel>
+
+# Privacy policy
+
+Turnip is a simple timer app made by Sarah &amp; Matt at [Octopus Think](https://octopusthink.com/about).
+
+By default, Turnip collects **absolutely no data** about you, your computer, or your usage of the app.
+
+## Beta programme
+
+If you sign up for the Turnip beta, crash reporting and analytics are mandatory. This is to ensure that we know if anything is going wrong. Your data is anonymised and isn’t tied to any personal identifier in any way. If that makes you uncomfortable, we totally understand, but we think this is the best way to ensure that we can improve the app for our beta users as quickly and efficiently as possible.
```

Comment:
So: in the future, we will likely end up associating a crash report with a user ID. This allows us to check out non-private user data associated with an account for crashes.

Right now there aren't any identifiers, but just something to think about...